packages/dashboard/src/utils/permissions.ts

```diff
@@ -0,0 +1,10 @@
+export async function extraPermissions(origins: string[] | null) {
+    if (!origins) return []
+    const currentOrigins = (await browser.permissions.getAll()).origins || []
```

Comment:
Please do not call `browser` API in dashboard.

Reply:
You may refer to `checkPermissionsAndRestore` in `WelcomeService`.

Reply:
ok